app/src/main/java/com/github/se/eventradar/model/repository/event/MockEventRepository.kt

```diff
@@ -75,4 +76,64 @@ class MockEventRepository : IEventRepository {
       }
     }
   }
+
+  override suspend fun addAttendee(eventId: String, attendeeUserId: String): Resource<Unit> {
+    return when (val res = getEvent(eventId)) {
+      is Resource.Success -> {
+        res.data?.attendeeList?.add(attendeeUserId)
```

Comment:
question: this has to be checked for null because getEvent returns Resource<Event?>, what do you think about changing the return type of getEvent to Resource<Event> (in both Mock and Firebase Repo) and doing the null checks for the data in getEvent, so that we don't have to use the `?` in `res.data?.attendeeList?.add(attendeeUserId)`? Maybe this is too much refactoring though but helps reduce the possibility of null event data being used in other functions

Reply:
Good point, I should ask the person who coded getEvent what they had in mind before making any changes though